### DISCUSSION TITLE
Add auto code completion﻿ for cp editor

### DISCUSSION THREAD

**Discussion by amrhossamdev** [cpeditor/cpeditor#1110] [Ideas]
Try to add auto code completion﻿ for the cp editor when writing functions...

![intellisense (1)](https://user-images.githubusercontent.com/29693819/199268961-c7078e1a-7e02-40bb-add8-a01efb9feff4.gif)

**Comment by ouuan** [comment]
#156

**Comment by Gr-Nobel** [comment]
![image](https://user-images.githubusercontent.com/95166364/203481547-4f159619-2339-4d0d-ac16-76258996d6ef.png)
![image](https://user-images.githubusercontent.com/95166364/203481565-7a407f94-8ae4-4315-8d9b-61580e198460.png)
![image](https://user-images.githubusercontent.com/95166364/203481583-40e4cad3-4f7e-4f1a-a9d3-e85c57b8aca9.png)
What I'm doing wrong? I have added LLVM bin to path also. no auto completion and the linter is showing red on well code. please help.

**Reply by ouuan** [reply]
Please open a new Discussion. Don't ask in unrelated threads.
